pkg/nodeps/values.go

```diff
@@ -41,6 +41,12 @@ var ValidOmitContainers = map[string]bool{
 	DBAContainer:          true,
 }
 
+// Router types
+const (
+	TraditionalRouter = "traditional"
```

Comment:
Can we name it by name? `nginx` or `nginx-proxy`. Traditional assumes some knowledge which we both have but not anyone else.

Reply:
Changed to nginx-proxy